test/redux/ideas_test.js

```diff
@@ -146,11 +146,12 @@ describe("actionCreators", () => {
 
         it("dispatches an error", () => {
           push = mockRetroChannel.push("anyEventJustNeedThePushInstance", { foo: "bar" })
+          push.trigger("ok", {})
```

Comment:
this is the only thing that gets me, as i can't understand *why* this is suddenly necessary. gonna peep through the phoenix source to see if there's something we can isolate in the `setupMockPhoenixChannel` helper

Reply:
I know - I decided to ship it after too much time in the rabbit hole.  Would be great if you could provide insight.

Reply:
yeah, i've been rabbit holing for awhile, and it's baby steps

Reply:
by which i mean, slow and unsteady progress